src/Compilers/CSharp/Test/Syntax/Parsing/SeparatedSyntaxListParsingTests.cs

```diff
@@ -39,7 +39,25 @@ class C
     A<,T> a7;
     A<T U,,> a8;
 }
-");
+",
+                // (7,9): error CS1003: Syntax error, ',' expected
+                //     A<T U> a4;
+                Diagnostic(ErrorCode.ERR_SyntaxError, "U").WithArguments(",").WithLocation(7, 9),
```

Comment:
perhaps the original authors of the tests didn't care about the errors, but it feels like this is a good thing. It feels like if someone wants to in the future they could always introduce a 'UsingTreeIgnoreErrors' method.